bin/uid_postgres.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Needs the license here...but also curious why this is in this patch and not with the original compaction

Reply:
Thats a good question. Ill look into it why and get it fixed